azext_iot/sdk/iothub/service/models/job_properties_py3.py

```diff
@@ -101,6 +104,7 @@ def __init__(self, *, job_id: str=None, start_time_utc=None, end_time_utc=None,
         self.output_blob_name = output_blob_name
```

Comment:
Please verify that the dataplane export + import jobs show up with the `az iot hub job` commands such as `list` and `show`.